modules/gitlab-runner/pvc.tf

```diff
@@ -0,0 +1,17 @@
+resource "kubernetes_persistent_volume_claim" "this" {
+  count = var.cache.type == "pvc" ? 1 : 0
+
+  metadata {
+    name      = "cache"
+    namespace = var.namespace
+  }
+  spec {
+    storage_class_name = "efs-sc"
```

Comment:
the "efs-sc" storage class hardcoding seems is not right as it is not possible to use another storage classes in other clusters than eks,
can we maybe have this as variable field under var.cache.storage_class_name with default value?